website/docs/feature-flag-tutorials/use-cases/trunk-based-development.md

```diff
@@ -0,0 +1,222 @@
+
+Unleash provides a powerful mechanism for safely managing and controlling these features in production, enabling enterprises to deliver software faster and with greater reliability. Effective feature flag management ensures that trunk-based development supports continuous delivery without compromising stability. In this tutorial, we’ll leverage Unleash to manage trunk-based development in your codebase.
+
+## How to Implement Trunk-Based Development with Feature Flags
+
+To follow along with this tutorial, you need access to an Unleash instance to create and manage feature flags. Head over to our [Quick Start documentation](/quickstart) for options, including running locally or using an [Unleash SaaS instance](https://www.getunleash.io/pricing?).
+
+With Unleash set up, you can use your application to talk to Unleash through one of our [SDKs](/reference/sdks).
+
+In this tutorial, you will:
+
+1. [Explore many trunk-based development strategies](#choose-the-right-trunk-based-development-strategy-for-your-team)
+2. [Create feature flags for incomplete features](#create-feature-flags-for-incomplete-features)
+3. [Control feature rollouts](#control-feature-rollouts)
+4. [Simplify rollbacks with Unleash](#simplify-rollbacks)
+5. [Archive and remove feature flags](#archive-and-remove-feature-flags)
+
+## Choose the Right Trunk-Based Development Strategy for Your Team
+
+There is no one-size-fits-all approach to implementing trunk-based development. Different teams may benefit from varying strategies based on their needs, project complexity, and development culture.
+
+In this section, we'll explore the common trunk-based development strategies and their trade-offs to help you choose the best approach for your organization.
+
+### Merge Every Commit
+
+The most frequent approach to trunk-based development is merging every commit directly into the main branch. This strategy maximizes the rate of continuous integration and delivery, as changes are integrated into the trunk as soon as they are ready. Some advantages are:
+
+-   Rapid feedback loops and quicker issue resolution
+-   Reduced risk of large, complex merges
+-   Encourages small, frequent commits
+
+Some challenges with merging every commit are:
+
+-   Potential for more frequent build breaks and test failures
+-   Increased need for effective merging strategies
+-   Requirement for robust automated testing and CI/CD pipelines
+
+### Merge at Set Intervals
```

Comment:
I've never heard of anyone advocating this. Not a challenge, just curious who's going this route, this sounds really painful!

Edit: I see Atlassian is recommending this, which explains why I think that's crazy